scRNA-seq-advanced/01-read_filter_normalize_scRNA.Rmd

```diff
@@ -208,14 +230,16 @@ filtered_sce
 
 ## Additional Quality control
 
-In addition to filtering out empty droplets, we also will want to filter out real cells that may have been damaged during library preparation.
+In addition to filtering out empty droplets, we also will want to filter out cells that may have been damaged during library preparation.
```

Comment:
Part of me thinks it might be helpful to state here that you also want to consider number of genes detected in a cell. I don't want them to get the impression that mitochondrial content is the only thing that should be considered. Even though you talk about how they can be correlated and how miQC filters both, I think this intro should address that both a low number of genes detected and high mito content indicate a compromised cell.

Reply:
Thanks for this suggestion, I added some clauses here to note that both of these features are important, and I think it motivates the model better!